### ISSUE TITLE
Build issue with Jekyll theme specified in _config.yml

### ISSUE THREAD

**Issue by tomaroberts** [GSTT-CSC/csc-test-site#26]
There is a small annoyance regarding specifying the Jekyll theme in the _config.yml, which I think is related to the csc-test-site repo not being in the default GitHub location (gstt-csc.github.io), which is where it will be once live.

Bit more explanation...

I started the new website based on the _Agency_ theme, which is not one of GitHub Pages official ones. In the Agency _config.yml the theme is specified on line 14: https://github.com/raviriley/agency-jekyll-theme/blob/master/_config.yml

In order to get the new CSC website working online at https://gstt-csc.github.io/csc-test-site/index.html I had to change to choose one of GitHub Pages official themes within the GitHub Pages Settings – see here: https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site. It's basically a hacky way of getting it to display online as GitHub Pages doesn't seem to recognise jekyll-agency

**TL;DR:**
In _`config.yml`
- for viewing on localhost, set `theme: jekyll-agency`
- for viewing online, set `theme: jekyll-theme-cayman`

**Comment by hshuaib90** [comment]
My theme is set to Jekyll-agency and I still can't bundle install .

**Comment by tomaroberts** [comment]
@hshuaib90 

> My theme is set to Jekyll-agency and I still can't bundle install .

Can you try removing `theme: jekyll-agency` and replace with `remote_theme: raviriley/agency-jekyll-theme` instead?

**Comment by hshuaib90** [comment]
Do I need to change the .gemspec ?

**Comment by Lucrezia-Cester** [comment]
Hi @hshuaib90, could you please attach a screenshot of the error you get when trying to bundle install  after you change the theme?

**Comment by AnilMistry** [comment]
@hshuaib90 I was able to get the website up and running. I use these commands each time I spin up the website on the local host, it may help solve your issue. (I'm not going to pretend I fully know what it does!)

   
cd Documents/GitHub/gstt-csc.github.io
rbenv global 3.0.1
env | grep PATH
rbenv global 3.0.1
export PATH="$HOME/.rbenv/bin:$PATH"
eval "$(rbenv init -)"
bundle install
bundle exec jekyll serve

**Comment by AnilMistry** [comment]
The install worked for me. I only needed to edit the _config.yml as above.

**Comment by tomaroberts** [comment]
Most of those commands are just to do with setting up your ruby environment by looks of it. Specifically looks like you are using ruby 3.0.1. The installation instructions I created install ruby using brew (package manager on MacOS).

Btw – you can run `bundle exec jekyll serve --livereload` if you want to see your website edits updated live. Note, if you make edits to the _config.yml, you'll need to cancel and re-run the line again.

**Comment by hshuaib90** [comment]
@tomaroberts  Changing to `remote_theme: raviriley/agency-jekyll-theme` or doing the commands that @AnilMistry recommended didn't work either. I'm going to see if nuking my Ruby set up might work.

**Comment by tomaroberts** [comment]
Btw – the latest advice on this is to ensure your `_config.yml` has:

`theme: jekyll-agency`

This should be sufficient to run the website on localhost.

Other issues are likely related to ruby/package problems. In this case, let me know and I'll have a look with you.

**Comment by tomaroberts** [comment]
No reports of problems running site locally for while. Keep issue open until go-live.

**Comment by tomaroberts** [comment]
Resolved... turns out you don't need `theme: ` in the `_config.yml`...
